src/core/remote.js

```diff
@@ -1852,8 +1856,15 @@ Remote.prototype.createPathFind = function(options, callback) {
   }
 
   if (callback) {
+    const updateTimeout = setTimeout(() => {
```

Comment:
I guess because the request timeout is per-response and we could keep getting responses without getting a full_reply?